bin/templates/cordova/lib/build.js

```diff
@@ -402,6 +440,16 @@ function parseOpts(options, resolvedTarget) {
                 case 'gradleArg':
                     ret.extraArgs.push(flagValue);
                     break;
+                case 'keystore':
+                case 'alias':
+                case 'keytorePassword':
+                case 'password':
+                case 'keystoreType':
+                    packageArgs[flagName] = flagValue;
+                    break;
+                case 'buildConfig':
+                    buildConfig = flagValue.replace(/"/g, '');
```

Comment:
That's true only when the argument is specified separately e.g. build --buildConfig "foo bar\build.json" - the shell will remove the quotes. But not for a scenario like --buildConfig="foo bar\build.json"